common/src/main/java/org/apache/celeborn/common/network/sasl/SaslUtils.java

```diff
@@ -35,17 +35,33 @@ public class SaslUtils {
   /** Quality of protection value that does not include encryption. */
   static final String QOP_AUTH = "auth";
 
+  /** Quality of protection value that includes encryption. */
+  static final String QOP_AUTH_CONF = "auth-conf";
```

Comment:
We are not relying on SASL to provide encryption support, why are we using `auth-conf` instead of `auth` ?

Reply:
Removed "auth-conf"